robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -1,6 +1,23 @@
 REQUEST_TIMEOUT = 3000
 ROTATE_TIMEOUT = 1000
+const PING_THROTTLE_TIME = 1000
 lastRotate = 0
+var lastCMDSent = 0
+
+// reset recorded time sent since last command
+function resetTimeSinceCMD() {
+  lastCMDSent = 0
+}
```

Comment:
Documentation is sometimes appreciated, but this comment is useless since the function name implies that very well. However, one reason why we should comment this function : what is the unit of time. ns? ms?

Reply:
The time is recorded in ms, so I decided to change the comment to 'reset time to 0ms'. I also modified the comment for `setTimeSinceCMD()` to make that clear.